### ISSUE TITLE
SA1125 UseShorthandForNullableTypes incorrectly reported in typeof()

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#386]
SA1125 is currently being reported for the following statement.

```csharp
Type nullableType = typeof(Nullable<>);
```

**Comment by sharwell** [comment]
This issue came back in CTP 6.
